sfge/src/main/java/com/salesforce/rules/UnusedInterfaceRule.java

```diff
@@ -13,18 +13,27 @@
 import org.apache.tinkerpop.gremlin.structure.Vertex;
 
 public class UnusedInterfaceRule extends AbstractStaticRule {
-    private static UnusedInterfaceRule INSTANCE;
+    private static final String URL =
+            "https://forcedotcom.github.io./sfdx-scanner/en/v3.x/salesforce-graph-engine/rules/#UnusedInterfaceRule";
+    private static final String DESCRIPTION =
```

Comment:
@teresa-allen-sfdc , description needs approval.

Reply:
Same as earlier. Can we move the text to UserFacingMessages.java?

Reply:
Probably.

Reply:
@jfeingold35 Thanks! So remember that I need to run the new rules plus related error message past UI text review. I'm finalizing the UI text review doc now, and expect to have comments back in a week.